frontend/src/components/status/StatusCheck.jsx

```diff
@@ -41,23 +23,23 @@ class StatusCheck extends React.PureComponent {
     const { description, isPassing } = this.props;
 
     return (
-      <div style={styles.container}>
-        <div style={styles.icon}>
+      <tr>
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
+              <i className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
             </div>
           )}
           {!isPassing && (
             <div style={styles.failing}>
-              <Glyphicon style={styles.glyphicon} glyph="remove-circle" />
-              <Label>{LOCALIZE.commons.failStatus}</Label>
+              <i className="fas fa-times-circle" />
+              <span>{LOCALIZE.commons.failStatus}</span>
```

Comment:
Same comment as my previous one, but for the fail status:

Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_, to have a little space between the check mark and the Fail word?